integrations/langflow/src/stepflow_langflow_integration/executor/langflow_server.py

```diff
@@ -17,13 +17,22 @@
 Clean architecture without CachedStepflowContext.
 """
 
+import logging
 from typing import Any
 
 from stepflow_py import StepflowContext, StepflowServer
 
 from .udf_executor import UDFExecutor
 
 
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    force=True  # Override any existing configuration
+)
+
+
 class StepflowLangflowServer:
```

Comment:
It feels like we shouldn't need this class given the pattern elsewhere, but probably OK?

Reply:
Well, it encapsulates StepflowServer and UDFExecutor spin up with the component registration (and the StepflowContext through register_componets). I agree this is a little squirrely and we need to straighten this out between integrations and python sdk.